src/main/java/uk/ac/ebi/eva/evaseqcol/model/NameLengthPairEntity.java

```diff
@@ -0,0 +1,25 @@
+package uk.ac.ebi.eva.evaseqcol.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+public class NameLengthPairEntity {
+    private String name;
+    private String length;
+
+    public NameLengthPairEntity(String name, String length) {
+        this.name = name;
+        this.length = length;
+    }
+
+    @Override
+    public String toString() {
+        return "{\n" +
```

Comment:
Is the modified string the one being hashed [above](https://github.com/EBIvariation/eva-seqcol/pull/30/files#diff-050649bcbe300e924f3143fad46fd2f088c0e992a52caba6ae306847be12e519R194)? In that case, I am not sure if we should be doing fancy newline additions...

Reply:
Just reviewed the seqCol-spec and you're correct. 
I did some tests and it reproduced the same digest with and without the newline/tabspace additions.
That should be removed.